sapphire/tests/esd_load_data.py

```diff
@@ -29,11 +32,35 @@ def perform_load_data(filename):
         esd.load_data(datafile, '/', weather_source, 'weather')
 
 
+def perform_esd_download_data(filename):
+    """Load data from esd/api to h5 (filename)"""
+
+    filters = tables.Filters(complevel=1)
+    start = datetime.datetime(2012, 01, 01, 0, 0, 0)
```

Comment:
Be careful with prepending numbers by a zero, this makes them octal.
For example try: `print 010`, this outputs 8, because octal 10 is 8.